app/Http/Controllers/Group/Workspace/HearingController.php

```diff
@@ -25,9 +25,7 @@ public function index(Group $group, Workspace $workspace, State $state, $year)
                             })
                             ->get()
                             ->sortBy(function ($history) {
-                                return $history->nebraska_hearing_date
-                                     ? $history->nebraska_hearing_date->diff(Carbon::now())->days
-                                     : null;
+                                return $history->nebraska_hearing_date?->diff(Carbon::now())->days ?? PHP_INT_MAX;
```

Comment:
I know you haven't tested this because `->days` isn't a thing. Also in what world would the fallback be `9223372036854775807` ??? I would think that either `null` or `32` (one more than the number of days in a month) would be perfectly sufficient, however, since you're sorting by a non-existent value in the first place, it's quite hard to tell how this is supposed to function in theory.

Reply:
It looks like `days` works, but the logic overall needed to be fixed.